src/Aspire.Hosting.Redis/api/Aspire.Hosting.Redis.cs

```diff
@@ -22,10 +22,14 @@ public static partial class RedisBuilderExtensions
 
         public static ApplicationModel.IResourceBuilder<Redis.RedisInsightResource> WithDataVolume(this ApplicationModel.IResourceBuilder<Redis.RedisInsightResource> builder, string? name = null) { throw null; }
 
+        public static ApplicationModel.IResourceBuilder<ApplicationModel.RedisResource> WithHostPort(this ApplicationModel.IResourceBuilder<ApplicationModel.RedisResource> builder, int port) { throw null; }
```

Comment:
```suggestion
        public static ApplicationModel.IResourceBuilder<ApplicationModel.RedisResource> WithHostPort(this ApplicationModel.IResourceBuilder<ApplicationModel.RedisResource> builder, int? port) { throw null; }
```

Everywhere else - `WithHostPort` accepts `null` except for Redis and SqlServer. The docs even say `If <see langword="null"/> is used random port will be assigned.`

Reply:
I'll fix that.

Reply:
#9282